browser/admin/src/integrator/AdminIntegratorSettings.ts

```diff
@@ -682,6 +682,32 @@ document.addEventListener('DOMContentLoaded', () => {
 		initTranslationStr();
 		(window as any).settingIframe = new SettingIframe();
 		(window as any).settingIframe.init();
+		const postHeight = () => {
+			window.parent.postMessage(
+				{
+					MessageId: 'Iframe_Height',
+					SendTime: Date.now(),
+					Values: {
+						ContentHeight: document.documentElement.offsetHeight + 'px',
+					},
+				},
+				'*',
+			);
+		};
+
+		let timeout: any;
+		const debouncePostHeight = () => {
+			clearTimeout(timeout);
+			timeout = setTimeout(postHeight, 100);
+		};
+
+		const mutationObserver = new MutationObserver(debouncePostHeight);
```

Comment:
yes, mutation shouldn't cause false alarms :)

Reply:
So when using a MutationObserver, it won’t detect changes caused by CSS class modifications, it only observes actual DOM changes.

That’s why I assume no one will change the iframe height via CSS alone, it would require a DOM change for the observer to catch it.